test/language/expressions/class/decorator/metadata/metadata-inheritance-not-affected-when-base-metadata-replaced.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2023 Ron Buckton. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-runtime-semantics-classdefinitionevaluation
+description: >
+    Metadata on a derived class inherits from the metadata of the declared super class.

```

Comment:
Suggestion, the description could be a bit more specific here, more like the title of the test: maybe something like "Metadata on a derived class inherits from the original metadata of the declared super class, unaffected by overwriting"